src/blas/backends/cublas/cublas_scope_handle_hipsycl.cpp

```diff
@@ -0,0 +1,72 @@
+#include "cublas_scope_handle_hipsycl.hpp"
+
+namespace oneapi {
+namespace mkl {
+namespace blas {
+namespace cublas {
+
+cublas_handle::~cublas_handle() noexcept(false) {
```

Comment:
This section is common with cublas_scope_handle.cpp. Maybe put them in a common header file?

Reply:
Changed the cublas_handle to a templated task, and moved it to a separate header `cublas_handle.hpp`